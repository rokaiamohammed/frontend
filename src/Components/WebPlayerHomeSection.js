import React, {Component} from 'react';
import CardList from './MoodCardList';
import '../Components/WebPlayerHome.css';

const WebPlayerHomeSection = ({title,des,type}) =>{
    return(
        <div className='web-home'>
            <h1 className='web-h1 pl3'><a className="web-a">{title}</a></h1>
            <h4 className='web-h4 pl3'>{des}
                <a className="web-seeAll">SEE ALL</a>
            </h4>
            <CardList Mood={type}/>
        </div>
    )
}

export default WebPlayerHomeSection;