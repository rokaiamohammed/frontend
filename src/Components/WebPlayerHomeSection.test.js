import React from 'react';
import { shallow, mount } from 'enzyme';
import WebPlayerHomeSection from './WebPlayerHomeSection';

const Section = [{
    title: 'Top hits',
    des: 'no description',
    type:'Mood'

  }]

it('WebPlayerHomeSection renders without crashing', () => {
  expect(shallow(<WebPlayerHomeSection title={Section.title} des={Section.des} type={Section.type} />)).toMatchSnapshot();
});