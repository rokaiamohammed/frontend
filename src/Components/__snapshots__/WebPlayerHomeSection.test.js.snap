// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WebPlayerHomeSection renders without crashing 1`] = `ShallowWrapper {}`;

exports[`WebPlayerHomeSectionList renders without crashing 1`] = `ShallowWrapper {}`;
